.github/workflows/Cabal-Linux.yml

```diff
@@ -0,0 +1,67 @@
+name: "Hackage, Cabal, Linux"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+  schedule:
+    - cron: "45 02 * * *"
+
+
+jobs:
+
+  build10:
+    name: GHC 8.10
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    strategy:
+      matrix:
+        packageRoots: [ ./ ]
+    defaults:
+      run:
+        working-directory: ${{ matrix.packageRoots }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - uses: actions/setup-haskell@v1.1
+        with:
+          ghc-version: "8.10"
+          # cabal-version: "3.0.0.0"
+      - name: Install additional system packages
+        run: sudo apt install libsodium-dev
+      #  2020-08-01: NOTE: Nix instantiate still needed for HNix tests
+      - name: Install Nix
+        uses: cachix/install-nix-action@v10
+      - run: cabal v2-configure --disable-optimization --enable-tests --enable-deterministic
+      - run: cabal v2-build
+      - run: cabal v2-test
+
+
+  build20:
```

Comment:
What does `build20` mean? If it's just a name, how about referencing the main property/properties of this job?, e.g. `ghc804`.

Reply:
Well, there is `name` field for it.

GitHub CI has number of design choises I am complettely puzzled about. Like they not follow basic YAML RFC and so metadata build IDs restricted, and there is separate name field, if they respected YAML RFC - that could be one field.

There is Name, there is already tracking of renaming needed, metadata IDs does not bear any useful information, so `name` shows real name and required ID field is just old-style unstractural numbering, as old BASIC had, with the step of 10 to have the ability to put builds in between without the need of renaming everything.

Reply:
Thanks for the explanation!